localstack/services/cloudformation/models/cloudformation.py

```diff
@@ -150,7 +150,9 @@ def generate_waitcondition_url(account_id: str, region_name: str, stack_name: st
     region = client.meta.region_name
 
     bucket = f"cloudformation-waitcondition-{region}"
-    key = arns.cloudformation_stack_arn(stack_name=stack_name)
+    key = arns.cloudformation_stack_arn(
+        stack_name=stack_name, stack_id="id-123", account_id=account_id, region_name=region_name
```

Comment:
The value for stack ID `id-123` is picked up from the existing default argument.

Reply:
sorry, just realized that the definition changed as well :) but maybe it makes sense then to use a short-uid here instead? I am not sure why it was hard-coded in the first place though 🤔 Do you have any background information maybe?

Reply:
> Do you have any background information maybe?

I don't know I'm afraid! I was hesitant to change it to `short_uid()` because I grepped it and saw it being used in some snapshots.

Reply:
I see. Maybe we can consult with @dominikschubert when he is back. 
Anyhow, this should not be blocking the PR.